py/desispec/brick.py

```diff
@@ -75,28 +75,30 @@ def __init__(self, bricksize=0.5):
 
     def brickname(self, ra, dec):
         """Return string name of brick that contains (ra, dec) [degrees]
-        
+
         Args:
             ra (float) : Right Ascension in degrees
             dec (float) : Declination in degrees
-            
+
         Returns:
             brick name string
         """
         inra, indec = ra, dec
         dec = np.atleast_1d(dec)
         ra = np.atleast_1d(ra)
         irow = ((dec+90.0+self._bricksize/2)/self._bricksize).astype(int)
-        names = list()
-        for i in range(len(ra)):
-            ncol = self._ncol_per_row[irow[i]]
-            j = int(ra[i]/360 * ncol)
-            names.append(self._brickname[irow[i]][j])
+
+        ncol = self._ncol_per_row[irow]
+        jj = (ra/360.0 * ncol).astype(int)
+        names = np.empty(len(ra), dtype='U8')
```

Comment:
That's correct.  The documentation from DECaLS is [here](http://legacysurvey.org/dr3/files) and screenshot below.  

<img width="1198" alt="screen shot 2017-02-13 at 8 17 17 pm" src="https://cloud.githubusercontent.com/assets/1431820/22910696/ba55ad20-f229-11e6-84a4-5559d5bd6b1c.png">